### DISCUSSION TITLE
如何用animate.css给dialog添加动画

### DISCUSSION THREAD

**Discussion by ghost** [element-plus/element-plus#10477] [Q&A] [answered]
我会给html中可见的元素添加动画，但是不知道如何给弹窗添加动画，求教

**Comment by ryuhangyeong** [comment]
Please provide a minimal example so we can help!

**Reply by ghost** [reply]
Has been resolved

**Comment by ghost** [comment, marked as answer]
<style>
.el-message-box{
  animation: zoomIn; /* referring directly to the animation's @keyframe declaration */
  animation-duration: 0.2s; /* don't forget to set a duration! */

  
}
</style>
